### ISSUE TITLE
Create new technique for new contrast formula (threshold value)

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#1213]
One approach to dealing with the updated colour contrast value is to add a new technique using the new value, making it a sufficient one for the contrast criteria.
https://www.w3.org/2020/07/14-ag-minutes.html#item07

**Comment by bruce-usab** [comment]
Deprecated formula is currently referenced in Note to glossary reference for relative luminance, see:
https://www.w3.org/TR/WCAG21/#dfn-relative-luminance

Three instances of :

> if RsRGB <= 0.03928 ...

Should be:

> if RsRGB <= 0.04545 ...

**Comment by alastc** [comment]
Hi @bruce-usab, I think the task was to create a technique (maybe based on [G18](https://www.w3.org/WAI/WCAG21/Techniques/general/G18)?) that would use the newer value. 

That could be copy-paste G18 into a Google doc and highlight the updates you make, or:
- Create a new branch (e.g. wcag22-contrast-technique)
- Duplicate techniques/general/G18.html to contrast-update.html
- Make the text changes.
- Create a PR from that branch.

**Comment by bruce-usab** [comment]
Thank you @alastc for that clear direction.  My comment above was mostly to distill for myself the specifics of the discrepancy.

**Comment by alastc** [comment]
Sorry if I was 'process-splaining'! I'm just running through issue statuses and working out where things are.

**Comment by bruce-usab** [comment]
No, that was helpful!

**Comment by bruce-usab** [comment]
I have created PR https://github.com/w3c/wcag/pull/1223 using the correct threshold value.

**Comment by Myndex** [comment]
> One approach to dealing with the updated colour contrast value is to add a new technique using the new value, making it a sufficient one for the contrast criteria.
> https://www.w3.org/2020/07/14-ag-minutes.html#item07

Hi @alastc Hi @bruce-usab 

DOH me, I just now saw this thread... but before I engage too much, it this just about fixing the intercept for the piecewise sRGB (0.04045) or is this about taking/modifying SAPC to replace the current WCAG 2.x contrast math in a "one step forward" kind of way?

Andy
"Color Me Obsessed...uh...with color"

**Comment by alastc** [comment]
Hi Andy,

That's just a WCAG 2.x update allowing for the use of the 'newer' value.

-Alastair
